src/responder/pam/pamsrv_p11.c

```diff
@@ -746,6 +748,14 @@ struct tevent_req *pam_check_cert_send(TALLOC_CTX *mem_ctx,
 
     /* extra_args are added in revers order */
     arg_c = 0;
+
+    chain_id = sss_chain_id_get();
+
+    if (chain_id > 0) {
```

Comment:
In `ipa_selinux.c::selinux_fork_child()` this is done unconditionally.
Both variants are fine, but approach feels inconsistent.

Reply:
No need for this conditional in the child processes, it is now removed.

Reply:
I meant parent processes here.

Reply:
I thought it was causing unit test failures, but I checked again and it is not an issue. Fixed now.